### PR TITLE
Adding multiple Statistical functions to paddle backend

### PR DESCRIPTION
**Pull request by KevinUnadkat**
Added prod from Statistical to paddle backend

### CODE REVIEW

**Comment by KevinUnadkat** [comment]
issue - prod #10868

**Comment by KevinUnadkat** [comment]
hy @MahmoudAshraf97 
pls review this file it is probably ready to merge

**Comment by KevinUnadkat** [comment]
hy 
@MahmoudAshraf97
pls review this file it is probably ready to merge

File: ivy/functional/backends/paddle/statistical.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def std(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.std(x, dim=axis, unbiased=False, keepdim=keepdims)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:75]: please review the input arguments and make sure that the output adheres to them

File: ivy/functional/backends/paddle/statistical.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def sum(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return torch.sum(input=x, dim=axis, dtype=dtype, keepdim=keepdims)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:87]: use paddle implementation, and check that the args and kwargs are correct

File: ivy/functional/backends/paddle/statistical.py [hunk 3]
```diff
@@ -96,7 +96,7 @@ def var(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.var(x, dim=axis, unbiased=False, keepdim=keepdims)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:99]: same comments as above

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

**Comment by KevinUnadkat** [comment]
@MahmoudAshraf97 
made the changes , everything seems good now

File: ivy/functional/backends/paddle/statistical.py [hunk 4]
```diff
@@ -72,7 +72,7 @@ def std(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.std(x, axis=axis, unbiased=True, keepdim=keepdims)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:75]: you need to implement the correction parameter, the unbiased parameter in paddle sets correction = 1, but it needs to be a float, check the torch backend implementation as it's the closest one to paddle

File: ivy/functional/backends/paddle/statistical.py [hunk 5]
```diff
@@ -96,7 +96,7 @@ def var(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.var(x, axis, unbiased=true, keepdim=keepdims)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:99]: same comment as std

File: ivy/functional/backends/paddle/statistical.py [hunk 6]
```diff
@@ -112,7 +112,7 @@ def cumprod(
     dtype: Optional[paddle.dtype] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.cumprod(x, dim=None, dtype=None, name=None)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:115]: you need to implement the `exclusive` and `reverse` arguments here

File: ivy/functional/backends/paddle/statistical.py [hunk 7]
```diff
@@ -124,12 +124,12 @@ def cumsum(
     dtype: Optional[paddle.dtype] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.cumsum(x, axis=axis, dtype=dtype)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:127]: same comment as `cumprod`

File: ivy/functional/backends/paddle/statistical.py [hunk 8]
```diff
@@ -124,12 +124,12 @@ def cumsum(
...
 
 
 def einsum(
     equation: str,
     *operands: paddle.Tensor,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.einsum(equation, *operands)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/statistical.py:135]: Please verify the results with another backend to make sure the implementation is correct

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

**Comment by MahmoudAshraf97** [comment]
I recommend you to read both Ivy documentation and paddle documentation when implementing a function in order to know the difference in implementation because some functions work differently, also check other backends implementations for reference

**Review by MahmoudAshraf97 — APPROVED** [review]

**Pull request merged**